ci/test

```diff
@@ -66,6 +66,13 @@ cucumber_authenticators_azure() {
 cucumber_authenticators_gcp() {
   : DOC - Runs Cucumber GCP Authenticator features
 
+  # Validate that token files are available
+    gcp_token_file="authn-gcp/tokens/gcp_token_valid"
+    if ! [ -f "$gcp_token_file" ]; then
+      echo "GCP token file ''$gcp_token_file' not found."
+      exit 1
+    fi
+
   _run_cucumber_tests authenticators_gcp "" "$(_get_gcp_env_args)"
```

Comment:
`_get_gcp_env_args` this function will server both GCE & GCF tests? if yes maybe we should write this in a comment, because what happening is:
we are relying gce token to fetch also GCF service account email and service account id values, and we are using those values in the cucumber tests

Reply:
we can set separate env vars